cloudinit/cmd/main.py

```diff
@@ -580,6 +585,7 @@ def main_modules(action_name, args):
     init = stages.Init(ds_deps=[], reporter=args.reporter)
     # Stage 1
     init.read_cfg(extract_fns(args))
+    log_ppid(init.distro)
```

Comment:
Need to move this down a bit too after `log.setup_logging` otherwise no logs are written to /var/log/cloud-init.log. To retain original behavior, we may want to log this just after the `welcome(... w_msg)` to log it during the boot stage block.